src/components/FormView.jsx

```diff
@@ -143,12 +150,9 @@ const FormView = ({
                           {...subblock}
                           name={name}
                           onChange={(field, value) =>
-                            onChangeFormData(
-                              subblock.id,
-                              field,
-                              value,
+                            onChangeFormData(subblock.id, field, value, [
```

Comment:
why this change?

Reply:
:thinking:

Reply:
I've checked it and it is not needed, I have removed those changes